charts/nginx-gateway-fabric/tmp/tmp-nginx-deployment.yaml

```diff
@@ -0,0 +1,169 @@
+# apiVersion: apps/v1
+# kind: Deployment
+# metadata:
+#   name: tmp-nginx-deployment
+#   namespace: {{ .Release.Namespace }}
+# spec:
+#   template:
+#     spec:
+#       initContainers:
+#       - name: init
```

Comment:
need a quick clarification -  we will be using these files when doing locally for now? until we have the complete split finished ?

Should we document a change a development process if there is?

Reply:
These will likely be changed and used as part of https://github.com/nginxinc/nginx-gateway-fabric/issues/2840. Since these deployment options are all temporary in the feature branch, we shouldn't need to update any developer process docs.

Reply:
okay thank you for the clarification.